pdf-ebinterface-frontend/src/main/resources/client/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "knock-dropzone": "^4.3.1",
```

Comment:
@MentorPK i think we can remove this dependency again, as you've written the drag/drop logic in plain JS.